### PR TITLE
Deploy cinder-volume on cinder group

### PR DESCRIPTION
**Pull request by jovial**
The default configuration is to deploy cinder-volume to the storage group. This change will mean that cinder will deployed to the controller.

### CODE REVIEW

File: etc/kayobe/kolla/inventory/overcloud-services.j2 [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+
+[vitrage-ml:children]
+vitrage
+
+# Blazar
+[blazar-api:children]
+blazar
+
+[blazar-manager:children]
+blazar
```
Comment (oneswig) [inline etc/kayobe/kolla/inventory/overcloud-services.j2]: Does it matter that this file is not newline-terminated?

**Review by oneswig — APPROVED** [review]
This is the result we want, in that the cinder-volume driver for Ceph need not be co-resident with Ceph itself.

File: etc/kayobe/kolla/inventory/overcloud-services.j2 (hunk 1 above)
Comment (jovial) [reply etc/kayobe/kolla/inventory/overcloud-services.j2]: Don't think it does. I lifted this verbatim from eod-kayobe-config.

File: etc/kayobe/kolla/inventory/overcloud-services.j2 [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+
+[ceph-osd:children]
+storage
+
+# Cinder
+[cinder-api:children]
+cinder
+
+[cinder-backup:children]
+storage
```
Comment (markgoddard) [inline etc/kayobe/kolla/inventory/overcloud-services.j2]: Should this one be on the controller too?

**Review by markgoddard — APPROVED** [review]

**Pull request merged**